prboom2/src/gl_main.c

```diff
@@ -2870,22 +2868,6 @@ static void gld_DrawItemsSortByTexture(GLDrawItemType itemtype)
   }
 }
 
-static int no_overlapped_sprites;
-static int C_DECL dicmp_sprite_by_pos(const void *a, const void *b)
```

Comment:
There's a table of comparison functions where this might belong, but it just uses `dicmp_sprite` twice.  Hmm...

Reply:
Seems to be fallout from a457290581

Note sure why the code was removed.